### PR TITLE
Task 22998 - Docs section: BCDR - SLAs

### PR DESCRIPTION
**Pull request by jegarg**

### CODE REVIEW

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[790accc](https://github.com/jegarg/cloud-adoption-framework/commits/790accc41c7154d56adfb052087e5f699f3ad6b9)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md](https://github.com/jegarg/cloud-adoption-framework/blob/main/docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md) | :warning:Warning | | [Details](#user-content-34d018cab3d0089f621f6d4720dcf2b5-5973eebe)

<a id="34d018cab3d0089f621f6d4720dcf2b5-5973eebe"></a>
### [docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md](https://github.com/jegarg/cloud-adoption-framework/blob/main/docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md)
 - **Line 119, Column 290**: **[Warning: hard-coded-locale - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/azure/virtual-desktop/data-locations#data-locations' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 119, Column 290**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/en-us/azure/virtual-desktop/data-locations#data-locations' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202212140430543705-1027/BuildReport?accessString=b06584545dcdf99e4654e3d9f6b376e7393eeb5d561e23a16cd8f426fce6c769).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by jegarg** [comment]
@danycontre Please review and let me know if any change needs to be done.

**Review by igorpag — APPROVED** [review]

**Comment by igorpag** [comment]
#sign-off

File: docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md [hunk 1]
```diff
@@ -113,6 +117,15 @@ Preventing data loss in critical user data is important. For backup protection,
 
 If Azure Virtual Desktop users need access to on-premises resources, it's critical that you consider high availability in the network infrastructure that's required to connect to the resources. Assess and evaluate the resiliency of your authentication infrastructure, and consider BCDR aspects for dependent applications and other resources. These considerations will help ensure availability in the secondary disaster recovery location.
 
+### Data Locations for Azure Virtual Desktop
+
+Azure Virtual Desktop stores various information for service objects, such as host pool names, application group names, workspace names, and user principal names. Data is categorized into different types, such as customer input, customer data, diagnostic data, and service-generated data. [See more](https://learn.microsoft.com/en-us/azure/virtual-desktop/data-locations#data-locations)
```
Comment (Court72) [inline docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md:122]:
```suggestion
Azure Virtual Desktop stores various information for service objects, such as host pool names, application group names, workspace names, and user principal names. Data is categorized into different types, such as customer input, customer data, diagnostic data, and service-generated data. For more information, see [Data locations for Azure Virtual Desktop](/azure/virtual-desktop/data-locations#data-locations).
```

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[587af11](https://github.com/jegarg/cloud-adoption-framework/commits/587af116725b9081fc8718ce54364c3937132d6c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md](https://github.com/jegarg/cloud-adoption-framework/blob/main/docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202212141554019189-1027/BuildReport?accessString=51d775b1c38ddd2378c4f35a781c234592d50a91dfca53a762e7227ac15038a5).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Pull request merged**